styleguide/derek/templates/thought-leadership-overview/_partials/social-links.ejs

```diff
@@ -0,0 +1,4 @@
+<a class="rsTl-social-twitter" href="https://twitter.com" target="_blank"></a>
+<a class="rsTl-social-linkedin" href="https://linkedin.com" target="_blank"></a>
+<a class="rsTl-social-facebook" href="https://facebook.com" target="_blank"></a>
+<a class="rsTl-social-youtube" href="https://youtube.com" target="_blank"></a> 
```

Comment:
Can you add `rel="noopener noreferrer"` to all `target="_blank"` links per https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/